src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.Color;
+import core.basesyntax.Figure;
+import core.basesyntax.Shapes;
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private final int randomizer = random.nextInt(Shapes.values().length) + 1;
```

Comment:
Please dont make this variable class level field, when you create an instance of FigureSupplier variable randomizer  will be initialized and its value will not change, and this means that we will return the same random figure